Objects/complexobject.c

```diff
@@ -85,11 +85,63 @@ _Py_c_neg(Py_complex a)
 }
 
 Py_complex
-_Py_c_prod(Py_complex a, Py_complex b)
+_Py_c_prod(Py_complex z, Py_complex w)
```

Comment:
Should be ok now.  This doesn't require a mixed-mode variant.

Yet, maybe I should change variable names back for consistency with the rest?  Current naming here follows to the C standard.

Reply:
@serhiy-storchaka what do you think about renaming of arguments?

I think mapping local variables "abcd" -> "tuvw" will hot hurt readability and then we can preserve original argument names.

Reply:
I am fine with both variants. Preserving names from the old code can make the diff smaller, but in this case the new code is too different from the old code. Using names from the C standard examples helps comparing our code with the original code. The latter looks more important here.